debian/bionic/foreman/rules

```diff
@@ -24,6 +25,7 @@ build:
 	/bin/rm -f db/schema.rb
 	/bin/sed -ri '1sXenv rubyXenv foreman-rubyX' bin/*
 	/bin/sed -ri 's~^BUNDLER_CMD=""~BUNDLER_CMD="/usr/bin/foreman-ruby /usr/bin/bundle exec"~' script/foreman-rake
+	/bin/sed -i 's#puma-status#cd /usr/share/foreman \&\& /usr/bin/bundle exec &#' script/foreman-puma-status
```

Comment:
Why is `&&` escaped? but `&` isn't? Is that some sed magic?

Reply:
`&` means "the match" in a regex, so `s/foo/bar &/` replaces "foo" with "bar foo".

and in the middle I want the literal `&&` for the shell. but also see https://github.com/theforeman/foreman/pull/8472#issuecomment-827548229